lib/assets/javascripts/views/org_admin/phases/edit.js

```diff
@@ -6,4 +6,19 @@ $(() => {
   // Attach handlers for the expand/collapse all accordions
   expandCollapseAll();
   Tinymce.init({ selector: '.phase' });
+
+  const parentSelector = '#sections_accordion';
+  $(parentSelector).on('ajax:before', (e) => {
+    const panelBody = $(e.target).parent().find('.panel-body');
+    return panelBody.attr('data-loaded') === 'false';
+  });
+  $(parentSelector).on('ajax:success', 'a[data-remote="true"]', (e, data) => {
+    const panelBody = $(e.target).parent().find('.panel-body');
+    panelBody.attr('data-loaded', 'true');
+    panelBody.html(data);
+    Tinymce.init({ selector: '.section' });
+  });
+  $(parentSelector).on('ajax:error', 'a[data-remote="true"]', () => {
+    // TODO something generic for every error
```

Comment:
probably good to add a generic error to the json collection of constants in 'application.rb' so that its translatable. Then display the message in the panelBody.